### ISSUE TITLE
Expeditions need to use weather

### ISSUE THREAD

**Issue by DogZeroX** [space-wizards/space-station-14#17385]
## Description

We got weather for this reason, make normal planets rain/storm, lavaland has the ash stuff
**Reproduction**

**Screenshots**

**Additional context**

**Comment by deltanedas** [comment]
theres already a framework for it but iirc planet weather was janky since roofs dont exist or something

**Comment by DogZeroX** [comment]
Not really? They work since it doesn't render on top of plating, only on grass and the like.

**Comment by deltanedas** [comment]
yes and theres grass etc in a few rooms which is really jarring
there might be station grass or something that blocks weather but im not sure

**Comment by Cheackraze** [comment]
It might look funny since the landing pad is steel tile too, so it wouldn't get the weather either

**Comment by deltanedas** [comment]
i dont think that matters as much since iirc landing pad is temporary until theres better way to navigate to 0,0

**Comment by SlamBamActionman** [comment]
Closing this as upcoming changes to Salvage (regardless of how they pan out) will make expeditions redundant.
